core/domain/translation_fetchers.py

```diff
@@ -43,3 +43,30 @@ def get_translation_from_model(translation_model):
         translation_model.target_language_code,
         translation_model.source_text,
         translation_model.translated_text)
+
+
+def get_translation_for_text(
```

Comment:
`get_machine_translation` might be clearer here.

Reply:
get_machine_translated_text perhaps? Should match the domain object type.

Having said that ... I just realized that the object name is quite unclear for this reason. It's a "text", so I was expecting this function to return a string and was quite surprised to find that it didn't. I wonder if it's worth renaming the model and corresponding domain object to MachineTranslation / AutogeneratedTranslation, which seems more like a "thing" than a string? Sorry for not catching this before.

Reply:
I changed this to `get_machine_translated_text`. I have to go for a bit though, I'll take another look at changing the model names later tonight.

Reply:
@seanlip I renamed these, PTAL.